tests/test_sdfq.sage

```diff
@@ -0,0 +1,80 @@
+from cryptographic_estimators.SDFqEstimator import SDFqEstimator
+from cryptographic_estimators.SDFqEstimator.SDFqAlgorithms import *
+from cryptographic_estimators.SDFqEstimator import SDFqProblem
+from math import  floor, log2, ceil, comb, comb as binomial, log2 as log
+
+
+load('tests/module/attack_cost.sage')
+load('tests/module/cost.sage')
+
+
+# global parameters
+params = {"bit_complexities": 0, "is_syndrome_zero": True, "nsolutions": 0}
+stern_params = {"bit_complexities": 1, "is_syndrome_zero": True, "nsolutions": 0}
+
+# correction term due to correction of the LeeBrickell procedure, see SDFqAlgorithms/leebrickell.py line 98/99
+def lee_brickell_correction(k):
+    return log(k, 2)*2 - log(binomial(k, 2), 2)
+
+
+def test_sdfq_LeeBrickell():
+    """
+    special value test for Lee-Brickell
+    """
+    # we need to subtract the difference of lee brickell
+    # and p = 1, because p = 2 is not always optimial
+    ranges = 0.01
+    n, k, w, q = 256, 128, 64, 251
+    t1 = log(ISD_COST(n, k, w, q), 2) + log(n,2) - lee_brickell_correction(k)
+    A = LeeBrickell(SDFqProblem(n, k, w, q, **params), **params)
+    t2 = A.time_complexity(p=2)
+    assert(t1 - ranges < t2 < t1 + ranges)
+
+    n, k, w, q = 961, 771, 48, 31
+    t1 = log(ISD_COST(n, k, w, q), 2) + log(n,2) - lee_brickell_correction(k)
+    A = LeeBrickell(SDFqProblem(n, k, w, q, **params), **params)
+    t2 = A.time_complexity(**{"p": 2})
+    assert(t1 - ranges < t2 < t1 + ranges)
+
+
+def test_sdfq_stern():
+    """
+    special value test for Stern
```

Comment:
Can we provide more details of what is tested here? So someone not very familiar with the literature can understand?